### PR TITLE
Videovard additional pattern

### PR DESCRIPTION
**Pull request by shellc0de27**
The "f" does not exist

### CODE REVIEW

File: lib/resolveurl/plugins/videovard.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 class VideoVardResolver(ResolveUrl):
     name = "videovard"
     domains = ["videovard.sx"]
-    pattern = r'(?://|\.)(videovard\.sx)/[vef]/([0-9a-zA-Z]+)'
+    pattern = r'(?://|\.)(videovard\.sx)/[ved]/([0-9a-zA-Z]+)'
```
Comment (Gujal00) [inline lib/resolveurl/plugins/videovard.py:28]: Why remove `f`? you could leave it in as `[defv]`

**Review by Gujal00 — CHANGES_REQUESTED** [review]

File: lib/resolveurl/plugins/videovard.py (hunk 1 above)
Comment (shellc0de27) [reply lib/resolveurl/plugins/videovard.py:28]: i removed it cause its not needed. tested a lot of embeds so not sure why its there in the first place. But i can put it back if you want

Comment (shellc0de27) [reply lib/resolveurl/plugins/videovard.py:28]: Doesn't seem like the site ever used an `f` at all. Do you want it back in there?

Comment (Gujal00) [reply lib/resolveurl/plugins/videovard.py:28]: okay we can take out `f` as per your change

**Comment by Gujal00** [comment]
Thanks

**Pull request merged**